concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -969,9 +972,13 @@ jobs:
                 mkdir generated-certificates
                 tar xzf cf-certs/cf-certs.tar.gz -C generated-certificates
 
-                terraform apply -var-file="paas-cf/terraform/((aws_account)).tfvars" \
+                cp cf-tfstate/cf.tfstate updated-tfstate/cf.tfstate
+                terraform init paas-cf/terraform/cloudfoundry
+                terraform apply \
+                  -var-file="paas-cf/terraform/((aws_account)).tfvars" \
```

Comment:
@samcrang It looks like we missed `--auto-approve=true` here. I might have broken it while rebasing. I can add it now if you're alright with me rebasing the branch?

Reply:
I'm okay with rebasing--rebasing is my favourite thing ever.

Reply:
Then have I got a force push for you! Fixed now.